File: js/CarPremium.js
class CarPremium extends Car {

  typeCarAccepted;
  seatsMaterial;

  constructor(license, driver, passenger, typeCarAccepted, seatsMaterial) {
    super(license, driver, passenger);
    this.typeCarAccepted = typeCarAccepted;
    this.seatsMaterial = seatsMaterial;
  }
}

class UberBlack extends CarPremium {

  constructor(license, driver, passenger, typeCarAccepted, seatsMaterial){
    super(license, driver, passenger, typeCarAccepted, seatsMaterial);
  }
}

class UberVan extends CarPremium {

  constructor(license, driver, passenger, typeCarAccepted, seatsMaterial){
    super(license, driver, passenger, typeCarAccepted, seatsMaterial);
  }
}

File: python/carPremium.py
from car import Car

class CarPremium(Car): 
  
  typeCarAccepted = []
  seatsMaterial = []
  
  def __init__(self, license, driver, passengers, typeCarAccepted, seatsMaterial):
    super(CarPremium, self).__init__(license, driver, passengers)
    self.typeCarAccepted = typeCarAccepted
    self.seatsMaterial = seatsMaterial
    
    
class UberBlack(CarPremium):
  def __init__(self, license, driver, passengers, typeCarAccepted, seatsMaterial):
    super(UberBlack, self).__init__(license, driver, passengers)
    self.typeCarAccepted = typeCarAccepted
    self.seatsMaterial = seatsMaterial
    
class UberVan(CarPremium):
  def __init__(self, license, driver, passengers, typeCarAccepted, seatsMaterial):
    super(UberVan, self).__init__(license, driver, passengers)
    self.typeCarAccepted = typeCarAccepted
    self.seatsMaterial = seatsMaterial